### PR TITLE
Add adjusted mkarchiso for loong64

### PR DESCRIPTION
**Pull request by YHStar**
Official "mkarchiso" doesn't support loong64 bootmodes, I adjusted mkarchiso add functions for loong64 bootmodes, now we can build iso easily 
```sudo ./mkarchiso (-v) .```
or maybe loong should make a brand_new script for isomake.

### CODE REVIEW

**Review by wszqkzqk — APPROVED** [review]
Good job!

File: pacman.conf [hunk 1]
```diff
@@ -32,10 +32,10 @@ Architecture = auto
 #UseSyslog
 #Color
 #NoProgressBar
-CheckSpace
+# We cannot check disk space from within a chroot environment
+#CheckSpace
 #VerbosePkgLists
-#ParallelDownloads = 5
-DownloadUser = alpm
```
Comment (wszqkzqk) [inline pacman.conf]: Why?

File: pacman.conf [hunk 2]
```diff
@@ -32,10 +32,10 @@ Architecture = auto
...
+ParallelDownloads = 5
```
Comment (wszqkzqk) [inline pacman.conf:38]: Why?

**Review by wszqkzqk — COMMENTED** [review]
All things not related to the architecture should comply with upstream.

File: pacman.conf (hunk 2 above)
Comment (starsaway) [reply pacman.conf:38]: revert, before simply cp from system , my bad

**Review by wszqkzqk — APPROVED** [review]

**Pull request merged**